plugins/main/public/controllers/overview/components/last-alerts-stat/last-alerts-stat.tsx

```diff
@@ -10,22 +16,87 @@ import {
   HttpError,
 } from '../../../../react-services/error-management';
 
-export function LastAlertsStat() {
+export function LastAlertsStat({ severity }: { severity: string }) {
   const [countLastAlerts, setCountLastAlerts] = useState<number | null>(null);
   const [discoverLocation, setDiscoverLocation] = useState<string>('');
+  const severityLabel = {
+    low: {
+      label: 'Low',
+      color: UI_COLOR_AGENT_STATUS.active,
```

Comment:
**suggestion**: maybe we should define other constants for these colors instead using the same for agent status connection.